### DISCUSSION TITLE
Broadcasting VoD playlist to YouTube is not working

### DISCUSSION THREAD

**Discussion by pngthis** [ant-media/Ant-Media-Server#6085] [Q&A]
Hello!

I set up an Ant Media server instance on DigitalOcean. 
Then I uploaded the Video under the VoD section of the Ant Media.

I got the Video URL from VoD, which is like this: `http://IP-Address:5080/LiveApp/streams/084369859678205487308525.mp4`

I added this VoD URL to the playlist 

Finally, I got the RTMP details from YouTube and I added them to playlist's Edit RTMP Endpoints.
The YouTube RTMP details look like this: `rtmp://a.rtmp.youtube.com/STREAM-KEY`

After adding everything correctly and despite the playlist broadcasting, YouTube doesn't capture the stream from Ant Media.

Please help me to understand how to fix this issue

Thank you so much

**Comment by USAMAWIZARD** [comment]
Can you please share the logs , You can collect the logs from /usr/local/antmedia/log directory via SSH into the server.
